lib/student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] The `parent_permission` parameter should be defined correctly in the super method to avoid errors. It uses colon for assignment so the way to assign it would be different from other parameters.

  ```
  super(age, name, parent_permission: parent_permission)
  ```

Reply:
Hi @DammyShittu thank you for the correction